com.github.sdbg.integration.jdt/src/com/github/sdbg/integration/jdt/JDTBreakpointPathResolver.java

```diff
@@ -33,6 +40,19 @@ public String getPath(IBreakpoint breakpoint) throws CoreException {
     }
   }
 
+  public String getLocalPath(IJavaBreakpoint bp) throws CoreException {
```

Comment:
The method does not need to be public. If so, please lower the visibility to 'private'.

Reply:
done